packages/experimental/Button/src/Button.types.ts

```diff
@@ -69,7 +69,6 @@ export interface ButtonProps extends Omit<IWithPressableOptions<ViewProps>, 'onP
    * A button can have its content and borders styled for greater emphasis or to be subtle.
    * - 'primary': Emphasizes the button as a primary action.
    * - 'subtle': Minimizes emphasis to blend into the background until hovered or focused.
-   * - 'transparent': NYI. Removes background and border styling.
```

Comment:
This is similar to our previous "ghost/stealth" button style, correct? I think that is what the sidebar button for FluentTester are implemented as, so expect a visual change whenever we eventually do make the visual changes

Reply:
No, ghost/stealth has been renamed to subtle, which has a transparent background but will color the background when the button is interacted with. Transparent will always have a transparent background - it changes the text color on interaction with the button.